app/scripts/modules/core/src/presentation/formsy/components/Input.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+
+import { FormComponent, IFormComponentState, IFormComponentProps } from '../FormComponent';
+
+export interface IProps extends IFormComponentProps {
+  className: string;
+  label: string;
+  type: string;
+}
+
+export interface IState extends IFormComponentState { }
```

Comment:
in this case, it needs the stuff from `IFormComponentState`, so I just switched it to that.

```
export class Input extends FormComponent<string, IProps, IFormComponentState> {
```